### DISCUSSION TITLE
Hello I bought a license to this mail and I can't activate

### DISCUSSION THREAD

**Discussion by juancarlosreyesp** [aclap-dev/video-downloadhelper#941] [License & Purchase requests]
### What's your issue?

Hello I bought a license to this mail and I can't activate on firefox I instaled coapp, 

Note. I reinstall frequently my computer and had activated in the same computer the same license.

### The 10 first digits (only 10) of your license number or email address

e203a9e782

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You should have received an email to reset your counter. Maybe in your spam box.
Anyway, I have reset your installation counter, should be OK now.

**Reply by juancarlosreyesp** [reply]
Thank you :)
